snapshot/snapshotter.go

```diff
@@ -17,27 +19,48 @@ const (
 	KindCommitted
 )
 
+var (
+	kindStrings = map[Kind]string{
+		KindView:      "View",
```

Comment:
Declare these as lower case. Print with `strings.Title` to get the nice output.

Reply:
This comment replaces your previous comment(` Use a switch, rather than a map here.`), right?